File: LeetCode/1281_Subtract_the_Product_and_Sum_of_Digits_of_an_Integer/solution.js
/**
 * @param {number} n
 * @return {number}
 */
var subtractProductAndSum = function(n) {
    let mult=1;
    let add=0;
    while(n>0){
        let a = n%10;
        mult*=a;
        add+=a;
        n-=a;
        n/=10;
    }
    return mult-add;
};

File: LeetCode/1281_Subtract_the_Product_and_Sum_of_Digits_of_an_Integer/solution.cpp
class Solution {
public:
    int subtractProductAndSum(int n) {
        int mult = 1;
        int add = 0;
        while (n > 0) {
            int a = n % 10;
            mult *= a;
            add += a;
            n -= a;
            n /= 10;
        }
        return mult - add;
    }
};